public/app/plugins/datasource/grafana-pyroscope-datasource/datasource.ts

```diff
@@ -101,6 +102,25 @@ export class PhlareDataSource extends DataSourceWithBackend<Query, PhlareDataSou
       labelMatchers: extractLabelMatchers(tokens),
     };
   }
+
+  getDefaultQuery(app: CoreApp): Partial<Query> {
+    return defaultQuery;
+  }
+}
+
+export const defaultQuery: Partial<Query> = {
```

Comment:
Just moved here as it seemed like better place and makes more straightforward imports.